src/core/models/directanswer.js

```diff
@@ -38,4 +38,28 @@ export default class DirectAnswer {
       }
     });
   }
+
+  /**
+   * Applies a formatter to the direct answer if a formatter exists for the direct answer field
+   *
+   * @param {Object.<string, function>} formatters
+   */
+  format (formatters) {
```

Comment:
would it make sense to call these format() methods within fromCore() instead of making them public?